ui/src/app/services/profile-storage.service.ts

```diff
@@ -49,17 +49,13 @@ export class ProfileStorageService {
     }
     this.activeCall = true;
 
-    // Conflict 409s are not retries automatically. These have a likelihood to
```

Comment:
Is this comment no longer true with your changes?

Reply:
A few parts to this one:

1. The rationale for this behavior is a bit outdated. In the past, we did a lot more lazy initialization with the profile API, and the client made many more parallel requests, so we saw lots of 409s on first login. We've untangled a lot of that logic, and this doesn't really happen anymore. I don't see this at all in test, and in prod in the past 21 days, there's only been one case of a 409 on the profile API (and this retry didn't help - it just 409'd 3 times in a row).
2. It is still true that 409s are not retried automatically (actually, with the fetch client, even 500s aren't being retried automatically, which is something we will want to address).
3. With this change, we're removing the 409 retries, which weren't actively bad, but also are no longer necessary. If we had a drop-in replacement, I might put it here.